### PR TITLE
Fix gubernator exception on empty system-out elements.

### PR DESCRIPTION
**Pull request by rmmh**
From #6092

### CODE REVIEW

File: gubernator/view_build.py [hunk 1]
```diff
@@ -52,10 +52,9 @@ def handle_test(self, child, filename, name_prefix=''):
         elif child.find('failure') is not None:
             time = float(child.attrib['time'])
             out = []
-            for param in child.findall('system-out'):
-                out.append(param.text)
-            for param in child.findall('system-err'):
-                out.append(param.text)
+            for param in child.findall('system-out') + child.findall('system-err'):
+                if param.text is not None:
+                    out.append(param.text)
```
Comment (fejta) [inline gubernator/view_build.py]: Are there any memory considerations here? Would something like `param.text[cut-out-middle]` make sense?

File: gubernator/view_build.py [hunk 2]
```diff
@@ -52,10 +52,9 @@ def handle_test(self, child, filename, name_prefix=''):
         elif child.find('failure') is not None:
             time = float(child.attrib['time'])
             out = []
-            for param in child.findall('system-out'):
-                out.append(param.text)
-            for param in child.findall('system-err'):
-                out.append(param.text)
+            for param in child.findall('system-out') + child.findall('system-err'):
+                if param.text is not None:
```
Comment (fejta) [inline gubernator/view_build.py]: How do `None` and `''` differ?

Comment (rmmh) [reply gubernator/view_build.py]: '\n'.join([None]) doesn't work

File: gubernator/view_build.py (hunk 1 above)
Comment (rmmh) [reply gubernator/view_build.py]: Not currently.

File: gubernator/view_build.py (hunk 2 above)
Comment (fejta) [reply gubernator/view_build.py]: Do we explicitly want to add empty strings?

**Pull request merged**